src/ColorContextMenu-Core.package/ColorChooserMorph.class/instance/getBlack..st

```diff
@@ -0,0 +1,4 @@
+accessing
+getBlack: aFraction
+
+	^ Color white ccmWithCmykBlack: aFraction
```

Comment:
To all the new get selectors: What about extracting these magic numbers (white and black) to something like `#referenceColorRgbForItems` and `#referenceColorCmyk`? Took me a while to find out where these values come from ^^

Reply:
done